suites/quincy/rbd/tier-2_rbd_regression_trash.yaml

```diff
@@ -0,0 +1,123 @@
+# Tier2: Extended RBD acceptance testing
+#
+# This test suite runs addition test scripts to evaluate the existing functionality of
```

Comment:
Can you be more specific on the tests being done in this test suite?

Reply:
Done.